index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Page</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/d6cd3c7604.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <!-- logo and navbar section -->
+    <header class="nav_container">
+        <div class="nav_logo">
+                <h2 class="nav_head">My Logo</h2>
+        </div>
+        <!--  -->
+        <nav class="toolbar">
+                <a class="nav_link" href="#">Portfolio</a>
+                <a class="nav_link" href="#">About</a>
+                <a class="nav_link" href="#">Contact</a>
+        </nav>
+    
+            <!-- RESPONSIVE NAVBAR -->
+        <nav class="mobile_menu">
+            <div class="my_logo"></div>
+            <div class="toggle_menu">
+                <img src="Union.png" alt="hamburger">
```

Comment:
- [ ] At the moment, the hamburger is not displaying. Please kindly make sure that the hamburger is displaying on the screen by downloading the right file.

![image](https://user-images.githubusercontent.com/43995830/164487786-e6a51c97-ab09-4e95-bbf4-d3d1ef8b1561.png)